vault-core/src/main/java/org/apache/jackrabbit/vault/packaging/impl/JcrPackageImpl.java

```diff
@@ -386,6 +386,7 @@ private void extract(Set<PackageId> processed, ImportOptions options, boolean cr
             // MAX_VALUE disables saving completely, therefore we have to use a lower value!
             opts.setAutoSaveThreshold(Integer.MAX_VALUE - 1);
         }
+        opts.setIdConflictPolicy(mgr.getDefaultIdConflictPolicy());
```

Comment:
This must be conditional, only override if not yet set!

Reply:
so that needs extra code in ImportOptions, right?

Reply:
I tried to embed the logic in ImportOptions, and extended "toString()" and friends accordingly. Now ExecutionPlanBuilderImplTest.testSaveAndLoad fails - what do I need to adjust?

Reply:
Changed to use null for not set. Please review.

Reply:
Line 392 must be removed!